src/client/Health/NewSouth.jsx

```diff
@@ -39,60 +45,223 @@ const NewSouth = () => {
           enabled: false,
         },
       })
+      setName('')
+      setProtocol(null)
+      handleClose()
     }
   }
 
   return (
     <>
       <Button
+        style={{ right: '0px' }}
         id="add-south"
-        className="inline-button autosize oi-south-button"
+        className="inline-button autosize oi-add-connector"
         size="sm"
-        outline
+        onClick={handleShow}
       >
         + South
       </Button>
-      <Popover
-        className="pop-container"
-        trigger="legacy"
-        placement="right"
-        target="add-south"
-        isOpen={openPopover}
-        toggle={togglePopover}
+
+      <Modal
+        show={show}
+        onHide={handleClose}
+        aria-labelledby="contained-modal-title-vcenter"
+        centered
+        size="lg"
       >
-        <PopoverBody>
-          <Form>
-            <Row>
-              <Col md="7">
-                <OIbText
-                  label="New DataSource Name"
-                  value={name}
-                  name="name"
-                  onChange={handleChange}
-                  defaultValue=""
-                  valid={() => validationSouth.protocol.isValidName(name, dataSources?.map((dataSource) => dataSource.name) ?? [])}
-                />
+        <Modal.Header className="oi-modal-header">
+          Sélectionner le protocole
+        </Modal.Header>
+
+        <Modal.Body>
+          <Container>
+            <Row style={{ marginTop: '25px' }}>
+              <Col xs={6} md={12}>
+                <div className="icon-container">
+                  <div className="icon-left-modal">
+                    <img src={iotSvg} alt="logo" height="24px" />
+                  </div>
+                  <div style={{ fontSize: '16px', paddingRight: '75px' }} className="icon-center ">
+                    IoT
+                  </div>
+                </div>
+
+                <div className="connector-container">
+                  <button
+                    id="icon-ADS"
+                    type="button"
+                    className={` ${(protocol === 'ADS' && active) ? 'connectorFocus ' : 'connector'}`}
```

Comment:
connectorFocus -> connector-focus
(CSS good practice is not camelCase)
Rename that everywhere